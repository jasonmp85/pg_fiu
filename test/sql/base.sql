\set ECHO 0
BEGIN;
\i sql/pg_fiu.sql
\set ECHO all

-- You should write your tests

SELECT pg_fiu('test');

ROLLBACK;
