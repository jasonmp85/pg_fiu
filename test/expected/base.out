\set ECHO 0
-- You should write your tests
SELECT pg_fiu('test');
 pg_fiu 
--------
 
(1 row)

ROLLBACK;
